swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, category = 'Clothing', condition = 0.0, age = 0):
+        super().__init__(category, condition, age = 0)
```

Comment:
Consider refactoring to pass `category='Clothing'` in the call to the `super` constructor. The affect this has is to make it so that when we instantiate a new clothing object, we don't have the option of passing in a different value for category. This feedback applies for Decor and Electronics too. Also note that since we set the default in our `def __init__` for condition and age, we don't need to do this again in our call to super

```suggestion
    def __init__(self, condition = 0.0, age = 0):
        super().__init__(category="Clothing", condition=condition, age=age)
```